src/FieldDescription/BaseFieldDescription.php

```diff
@@ -32,33 +32,31 @@
  * Some options are global across the different contexts, other are
  * context specifics.
  *
- * Global options :
- *   - type (m): define the field type (use to tweak the form or the list)
- *   - template (o) : the template used to render the field
- *   - name (o) : the name used (label in the form, title in the list)
- *   - link_parameters (o) : add link parameter to the related Admin class when
```

Comment:
Apparently `link_parameters` it us used in the route generator, so I guess is kind of global: https://github.com/sonata-project/SonataAdminBundle/blob/137ffd9f6053fa0b734c2e1dd7db86d6c3b94aed/src/Route/DefaultRouteGenerator.php#L98